governance.md

```diff
@@ -160,7 +160,7 @@ See [user group governance] for more details about forming and disbanding
 User Groups.
 
 To facilitate discoverability and engagement,
-user groups are documented in [sigs.yaml]
+user groups are documented in [sigs.yaml].
```

Comment:
Found that all the sentences in this post ends with `.` so made the sentences consistent